bigtop-deploy/puppet/modules/ambari/manifests/init.pp

```diff
@@ -23,6 +23,39 @@
   }
 
   class server {
+    #install python2 and init postgresql for openEuler
+    if $operatingsystem == 'openEuler' {
+      exec { "download_python2.7":
```

Comment:
Python 2.7 would be installed in Bigtop_toolchain: https://github.com/apache/bigtop/blob/openEuler-support/bigtop_toolchain/manifests/packages.pp#L395

Please remove the python-2.7 installation from here.

Reply:
> Python 2.7 would be installed in Bigtop_toolchain: https://github.com/apache/bigtop/blob/openEuler-support/bigtop_toolchain/manifests/packages.pp#L395
> 
> Please remove the python-2.7 installation from here.

1，Python 2.7 is alrady installed in Bigtop_toolchain. 
2.   The openeuler was not support python2,and can not find in rpm-package-management.so we remove python2 require in openeuler. 
![image](https://github.com/apache/bigtop/assets/31821890/f0f1a19a-353a-4974-a294-c4372b9d7035)
3.  when we use bigtop/puppet to run smoketest,can not find python2 in openeuler,and we can not to exec ambari-start or  ambari-agent. so we need to instadlled python in openeuler environment,and should add  this python code inbigtop-deploy/puppet/modules/ambari/manifests/init.pp
@guyuqi

Reply:
I think python is the fundamental tools.
What about moving python2 installation into `bigtop/puppet` for openEuler?

Reply:
> I think python is a fundamental tool. What about moving python2 installation into `bigtop/puppet` for openEuler?

The installation of python2 has moved into `bigtop/puppet`, the code commit:
f43bfb82391971fc0255b54f68c11da3ed59bf3a

@guyuqi

Reply:
Ambari smoke test successlly using bigtop/puppet, the log is:
![image](https://github.com/apache/bigtop/assets/11262916/b416b6a5-75ea-4729-9d19-1f3651cbd284)

The detail log is:
https://gitee.com/macchen1/bigtop/blob/master/smoke-test-log/openeuler-puppet-smoke-log/ambari-openeuler.log

@guyuqi 
We use the bigtop/puppet to smoke test is ok, please review the last PRs for openEuler.

**BIGTOP-3988: using bigtop/puppet for hadoop:**
https://github.com/apache/bigtop/pull/1168
**BIGTOP-3824: add support for ranger**
https://github.com/apache/bigtop/pull/1143
**BIGTOP-3967: add support for spark**
https://github.com/apache/bigtop/pull/1148

We can verfiy all components if you check and merge the PRs into openEuler-support branch, and we can fix the problem if it happen basing the branch.
Thanks.